internal/jimm/juju/jimm.go

```diff
@@ -64,15 +66,19 @@ func (j *JujuManager) ControllerInfo(ctx context.Context, name string) (*dbmodel
 	return &ctl, nil
 }
 
-// ListControllers returns a list of controllers the user has access to.
+// ListControllers returns a list of controllers the user has can_addmodel to.
+// JIMM admins get all controllers.
 func (j *JujuManager) ListControllers(ctx context.Context, user *openfga.User) ([]dbmodel.Controller, error) {
-	if !user.JimmAdmin {
-		return nil, errors.E(errors.CodeUnauthorized, "unauthorized")
-	}
-
 	var controllers []dbmodel.Controller
 	err := j.Database.ForEachController(ctx, func(c *dbmodel.Controller) error {
-		controllers = append(controllers, *c)
+		canAddModel, err := user.IsAllowedAddModelToController(ctx, c.ResourceTag())
```

Comment:
i will assume this returns true if everybody has this permission on the controller